client/valor/schemas/symbolic/collections.py

```diff
@@ -136,7 +143,8 @@ def decode_value(cls, value: dict):
     def encode_value(self):
         """Encode object to JSON compatible dictionary."""
         return {
-            k: v.encode_value() for k, v in self._get_dynamic_values().items()
+            k: (v.encode_value() if hasattr(v, "encode_value") else v)
+            for k, v in self._get_dynamic_values().items()
```

Comment:
I think this can be completely reverted to the original 

```
k: v.encode_value() for k, v in self._get_dynamic_values().items()
```

as all symbolic types have the encode_value member